tests/test_compile.py

```diff
@@ -14,7 +14,12 @@ class MyCompiler(Compiler):
 
 
 def test_initializes_empty_lists(snapshot: SnapshotTest):
-    out_src = Compiler().parse("tests.fixtures.default_values").to_zod()
+    out_src = Compiler().parse("tests.fixtures.default_values_list").to_zod()
+    snapshot.assert_match(out_src)
+
+
+def test_initializes_empty_dicts(snapshot: SnapshotTest):
+    out_src = Compiler().parse("tests.fixtures.default_values_dict").to_zod()
```

Comment:
I think you forgot to commit this file.
How about we keep everything in one file `default_values.py`?

Reply:
I forgot to add the files before committing :facepalm: 

Anyhow, I've pushed them now, I went for the split because I think that in general we want to avoid testing too much in a single test.